src/css/components/modal.scss

```diff
@@ -0,0 +1,20 @@
+
+@import "../override_vars.scss";
+@import "../core.scss";
+
+// faux modal for now, but maybe real later?
+
+.actions {
+  padding: 3rem 2rem;
+  background-color: #DEEEEF; // DEEF!
```

Comment:
Can we add this to `override_vars.scss`? Or use an existing color (preferred)? It can a color straight from the [standards](https://standards.usa.gov/visual-style/#colors).